foreman/client.py

```diff
@@ -222,7 +247,8 @@ def __init__(self, foreman):
         """
         self._f = foreman
         # Preserve backward compatibility with old interface
-        for method_name in ('index', 'show', 'update', 'destroy', 'create'):
+        for method_name in ('index', 'show', 'update', 'destroy', 'create',
+                            'search'):
```

Comment:
in my opinion the 'search' is duplicate to Foreman.<resource>.index(search="query"). could you please elaborate more what is purpose of this call ?